client/lib/transformer.dart

```diff
@@ -154,3 +162,5 @@ const _ANALYTICS_CODE = '''<script>
   ga('send', 'pageview');
 
 </script>''';
+
+const _SURVEY_CODE = '<script async="" defer="" src="//survey.g.doubleclick.net/async_survey?site=4vq7qmqpfn4zjduve64c4h6lqa"></script>';
```

Comment:
Is this encoding our site directly into the code? What happens on dartdocs.org? What happens on Angular docs, or some other site that's using this?